packages/engine/Source/Scene/Model/Model.js

```diff
@@ -1772,6 +1772,35 @@ Model.prototype.applyArticulations = function () {
   this._sceneGraph.applyArticulations();
 };
 
+/**
+ * Returns the object that was created for the given extension.
+ *
+ * The given name may be the name of a glTF extension, like `"EXT_example_extension"`.
+ * If the specified extension was present in the root of the underlying glTF asset,
+ * and a loder for the specified extension has processed the extension data, then
+ * this will return the model representation of the extension.
+ *
+ * @param {string} extensionName The name of the extension
+ * @returns The object, or `undefined`
```

Comment:
The object that is returned there is of the type that the object has _after_ parsing it from the extension JSON. In this case, it will have the type `GltfGpmLocal`.